### PR TITLE
Base Class

### PR DESCRIPTION
**Pull request by tokomon**

### CODE REVIEW

File: project/src/main/java/is2/Scraper.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package is2;
```
Comment (JcCcari) [inline project/src/main/java/is2/Scraper.java]: - reformate code with google code style

**Comment by JcCcari** [comment]
- move into a module.

File: project/src/main/java/is2/Scraper.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+public interface Scraper{
+	@Autowired
+	private String url;
+	//link
+	
+
+	public Collection<String> ParseData();
+		//maybe List<Product>
+	
+	public Collection<String> SaveData();
```
Comment (MathHellscream) [inline project/src/main/java/is2/Scraper.java]: This must pass as argument a Collection<Products>
and
This must not return anything(I suggest void)

File: project/src/main/java/is2/Scraper.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	public Collection<String> GetData();
```
Comment (hjneyra) [inline project/src/main/java/is2/Scraper.java]: Usar notación Camel

File: project/src/main/java/is2/Scraper.java (hunk 1 above)
Comment (tokomon) [reply project/src/main/java/is2/Scraper.java]: ok :dagger:

File: project/webScraper/src/main/java/domain/Scraper.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package is2;
+
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+public interface Scraper {
+
+  @Autowired
+  private String url;
```
Comment (hjneyra) [inline project/webScraper/src/main/java/domain/Scraper.java]: En una interface solo se definen método y atributos de clase (o estáticos)

File: project/webScraper/src/main/java/domain/Product.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package domain;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "product")
+public class Product {
```
Comment (hjneyra) [inline project/webScraper/src/main/java/domain/Product.java]: Agregar los métodos getter y setters

File: project/webScraper/src/main/java/domain/Shop.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "shop")
+public class Shop {
```
Comment (hjneyra) [inline project/webScraper/src/main/java/domain/Shop.java]: Agregar getter y setter de los atributos.

File: project/webScraper/src/main/java/domain/Scraper.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package is2;
```
Comment (hjneyra) [inline project/webScraper/src/main/java/domain/Scraper.java]: el paquete `is2` no coincide con  `domain`

File: project/webScraper/src/main/java/domain/Product.java (hunk 5 above)
Comment (JcCcari) [reply project/webScraper/src/main/java/domain/Product.java]: ok!

File: project/webScraper/src/main/java/domain/Shop.java (hunk 6 above)
Comment (JcCcari) [reply project/webScraper/src/main/java/domain/Shop.java]: ok!

File: project/src/main/resources/application.properties [hunk 8]
```diff
@@ -1,8 +1,8 @@
 spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:postgresql://localhost:5432/DATA_BASE
+spring.datasource.url=jdbc:postgresql://localhost:5432/is2_t1
 
-spring.datasource.username=USERNAME
-spring.datasource.password=PASSWORD
+spring.datasource.username=postgres
+spring.datasource.password=postgres
 
 # Don't push the modified version of this file!
```
Comment (Fredy) [inline project/src/main/resources/application.properties]: Please, dont push your modified version of aplication.properties

File: project/webScraper/src/main/java/domain/Scraper.java [hunk 9]
```diff
@@ -1,17 +1,14 @@
...
 import java.util.Collection;
 
-import org.springframework.beans.factory.annotation.Autowired;
-
 public interface Scraper {
 
-  @Autowired
-  private String url;
-
   public Collection<Product> parseData();
```
Comment (Fredy) [inline project/webScraper/src/main/java/domain/Scraper.java]: Base Class should let scrape Store data, and save it in the DB

**Pull request merged**